python/dask_cudf/dask_cudf/io/orc.py

```diff
@@ -115,12 +115,28 @@ def read_orc(path, columns=None, filters=None, storage_options=None, **kwargs):
     return dd.core.new_dd_object(dsk, name, meta, divisions)
 
 
-def write_orc_partition(df, path, fs, filename, compression=None):
+def write_orc_partition(
+    df,
+    path,
+    fs,
+    filename,
+    compression,
```

Comment:
Are we safe to remove the default value for this argument?

Reply:
There is no default value here now.